.github/workflows/build_and_test.yml

```diff
@@ -312,12 +312,11 @@ jobs:
       if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
       timeout-minutes: 20
       run: |
-        pip install poetry==1.6.1
+        pip install poetry==1.8.1
+        poetry config virtualenvs.create false
```

Comment:
Here I'm not using the default `.venv` with poetry and instead using the editable wheel installed earlier in the workflow